src/uniprotkb/components/entry/EntryPublications.tsx

```diff
@@ -92,7 +93,17 @@ const PublicationReference: FC<{ reference: Reference; accession: string }> = ({
     },
     {
       title: 'Cited for',
-      content: referencePositions?.join(', '),
+      content:
+        referencePositions &&
+        addBlastLinksToFreeText(referencePositions, accession).map(
+          (item, i) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <Fragment key={i}>
+              {i > 0 && ', '}
```

Comment:
You could use `listFormat` to handle this kind of conjunction in a proper sentence

Reply:
I think a comma separated list works better there than separating the blocks by OR 
![image](https://user-images.githubusercontent.com/3329765/145620195-53c7ed18-23e8-4ee3-ada1-e7441fe491e1.png)

Reply:
* AND